docs/reference/workspace-files.md

```
@@ -114,7 +114,7 @@ skills:
 
 ## workspace/projects.yaml
 
-Project registry. Read by `RouterPlugin` (GitHub enrichment), `GoalEvaluatorPlugin`, and `WorldStateEngine` (per-project domain discovery).
+Project registry. Read by `RouterPlugin` (GitHub enrichment), ``, and `` (per-project domain discovery).
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)